src/patient-registration/patient-registration.component.tsx

```diff
@@ -256,18 +357,21 @@ export const PatientRegistration: React.FC = () => {
         {props => (
           <Form className={styles.form}>
             <div className={styles.formTitle}>
-              <h1 className={`omrs-type-title-1 ${styles.title}`}>New Patient</h1>
-              {localStorage.getItem('openmrs:devtools') === 'true' && <DummyDataInput setValues={props.setValues} />}
+              <h1 className={`omrs-type-title-1 ${styles.title}`}>{!!existingPatient ? 'Edit' : 'New'} Patient</h1>
+              {localStorage.getItem('openmrs:devtools') === 'true' && !existingPatient && (
+                <DummyDataInput setValues={props.setValues} />
+              )}
             </div>
             <DemographicsSection setFieldValue={props.setFieldValue} values={props.values} />
             <ContactInfoSection addressTemplate={addressTemplate} />
             <IdentifierSection
               identifierTypes={identifierTypes}
               validationSchema={validationSchema}
               setValidationSchema={setValidationSchema}
+              inEditMode={!!existingPatient}
             />
             <button className={`omrs-btn omrs-filled-action ${styles.submit}`} type="submit">
-              Register Patient
+              {!!existingPatient ? 'Save Patient' : 'Register Patient'}
```

Comment:
Same here. Why the `!!`

Reply:
Even in cases where a boolean cast is actually necessary (which is quite rare), I think we should prefer the `Boolean(val)` syntax. Much clearer. `!!` is for code golf.